07-routing-gossip.md

```diff
@@ -565,6 +544,11 @@ the channel when the peer reestablishes contact.  Because gossip
 messages are batched and replace previous ones, the result may be a
 single seemingly-redundant update.
 
+The `must_be_one` field field in `message_flags` was previously used to
```

Comment:
```suggestion
The `must_be_one` field in `message_flags` was previously used to
```

Reply:
drive by change.

Reply:
Good catch, thanks! Rebased and fixed in https://github.com/lightning/bolts/pull/999/commits/0d10541b73d97ed5f56b02564830048506e060ab